core/fieldMapper.py

```diff
@@ -38,25 +40,49 @@ def _invoiceMapper(invoice_data):
             }
         ]
     }
+    if not is_manual:
+        return_dict['TxnTaxDetail'] = _get_tax_details(invoice_data.get('SALE TAX'), invoice_data.get('INVOICE ID'))
+
+    return return_dict
+
 
 def _customer_ref(cust_name, tv_invoice_id):
     result = queryCustomer(cust_name)
     if 'Customer' in result:
-       return {'value': result['Customer']['Id']}
+        return {'value': result['Customer']['Id']}
     else:
-        logger.error('error finding customer: {0} in Quickbooks while processing trackvia invoice: {1}'.format(cust_name, tv_invoice_id))
-        send_email('TV-QBO integeration error', 'We got an error finding customer: {0} in Quickbooks while processing trackvia invoice: {1}. Invoice creation/updation failed. Please create customer in quickbooks and retry.'.format(cust_name, tv_invoice_id))
+        logger.error(
+            'error finding customer: {0} in Quickbooks while processing trackvia invoice: {1}'.format(cust_name,
+                                                                                                      tv_invoice_id))
+        send_email('TV-QBO integeration error',
```

Comment:
no way of knowing whether it is manual or normal invoice.

Reply:
fixed